src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const trs = document.querySelectorAll('tr');
+
+trs.forEach((el) => {
+  const parentElement = el;
+  const firstChild = parentElement.children[1];
```

Comment:
The code is attempting to clone the second child of each table row. Ensure that the table rows have at least two children, otherwise this will result in an error.